sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,682 +3,526 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
-import { AmqpMessage } from "@azure/core-amqp";
-import { delay } from "@azure/core-amqp";
-import { Delivery } from "rhea-promise";
-import { HttpOperationResponse } from "@azure/core-http";
-import Long from "long";
-import { MessagingError } from "@azure/core-amqp";
-import { OperationOptions } from "@azure/core-http";
-import { PagedAsyncIterableIterator } from "@azure/core-paging";
-import { PageSettings } from "@azure/core-paging";
-import { PipelineOptions } from "@azure/core-http";
-import { RetryOptions } from "@azure/core-amqp";
-import { ServiceClient } from "@azure/core-http";
-import { TokenCredential } from "@azure/core-amqp";
-import { TokenType } from "@azure/core-amqp";
-import { UserAgentOptions } from "@azure/core-http";
-import { WebSocketImpl } from "rhea-promise";
-import { WebSocketOptions } from "@azure/core-amqp";
+
+import { AmqpMessage } from '@azure/core-amqp';
+import { delay } from '@azure/core-amqp';
+import { Delivery } from 'rhea-promise';
+import { HttpOperationResponse } from '@azure/core-http';
+import Long from 'long';
+import { MessagingError } from '@azure/core-amqp';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-http';
+import { RetryOptions } from '@azure/core-amqp';
+import { ServiceClient } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { UserAgentOptions } from '@azure/core-http';
+import { WebSocketImpl } from 'rhea-promise';
+import { WebSocketOptions } from '@azure/core-amqp';
 
 // @public
 export type AuthorizationRule = {
-  claimType: string;
-  claimValue: string;
-  rights: {
-    accessRights?: string[];
-  };
-  keyName: string;
-  primaryKey?: string;
-  secondaryKey?: string;
+    claimType: string;
+    claimValue: string;
+    rights: {
+        accessRights?: string[];
+    };
+    keyName: string;
+    primaryKey?: string;
+    secondaryKey?: string;
 };
 
 // @public
 export interface CorrelationRuleFilter {
-  contentType?: string;
-  correlationId?: string;
-  label?: string;
-  messageId?: string;
-  properties?: any;
-  replyTo?: string;
-  replyToSessionId?: string;
-  sessionId?: string;
-  to?: string;
+    contentType?: string;
+    correlationId?: string;
+    label?: string;
+    messageId?: string;
+    properties?: any;
+    replyTo?: string;
+    replyToSessionId?: string;
+    sessionId?: string;
+    to?: string;
 }
 
 // @public
 export interface CreateBatchOptions extends OperationOptionsBase {
-  maxSizeInBytes?: number;
+    maxSizeInBytes?: number;
```

Comment:
Wait! Why is this happening!!!

Reply:
We'll get a prettier invocation as a PR after this one. It's possible this is just annoying interaction with how I have my setup running again.